activerecord/lib/active_record/connection_adapters/postgresql/oid/uuid.rb

```diff
@@ -14,7 +14,9 @@ def type
           end
 
           def cast(value)
-            value.to_s[ACCEPTABLE_UUID, 0]
+            return nil unless value.is_a?(String)
+            return value.presence if value.match?(ACCEPTABLE_UUID)
```

Comment:
Why call `presence`? if you know the value will match `ACCEPTABLE_UUID` it can't be blank, or am I missing something?

Reply:
Good point. I had done it as homage to the original functionality of `UUID#cast` but unless there's a form of `ACCEPTABLE_UUID` which is also blank, then it won't be relevant.